decidim-comments/app/commands/decidim/comments/create_comment.rb

```diff
@@ -46,6 +46,8 @@ def create_comment
 
         mentioned_users = parsed.metadata[:user].users
         send_mention_notifications(mentioned_users) if mentioned_users.any?
+        linked_proposals = parsed.metadata[:proposal].linked_proposals
```

Comment:
Since this will keep growing when we add different kind of links I'd like to do it different. Instead of adding all the logic here, I'd broadcast the comment and its metadata with ActiveSupport::Notifications and then let each engine decide what to do with their linked data (decidim-proposals would send the events for proposals, decidim-accountability for results, etc..).

Example:

https://github.com/decidim/decidim/blob/master/decidim-core/app/services/decidim/settings_change.rb

https://github.com/decidim/decidim/blob/master/decidim-participatory_processes/app/commands/decidim/participatory_processes/admin/activate_participatory_process_step.rb#L69-L73

https://github.com/decidim/decidim/blob/master/decidim-proposals/lib/decidim/proposals/engine.rb#L81-L88

https://github.com/decidim/decidim/blob/master/decidim-proposals/app/jobs/decidim/proposals/settings_change_job.rb

Reply:
OK, I'll rework it for proposals.